packages/blockchain-wallet-v4-frontend/src/modals/Swap/PreviewSwap/index.tsx

```diff
@@ -29,6 +29,14 @@ class PreviewSwap extends PureComponent<InjectedFormProps<{}, Props> & Props> {
     this.props.swapActions.createOrder()
   }
 
+  networkFee = value => {
+    return value
+      ? value.coin === 'BTC' || value.coin === 'BCH'
```

Comment:
we should create one global enum for coin names and use it on every place we do have such checks